src/covid_model_seiir_pipeline/regression/task.py

```diff
@@ -97,7 +105,7 @@ def parse_arguments(argstr: Optional[str] = None) -> Namespace:
 
 
 def main():
-
+    configure_logging_to_terminal(verbose=1)  # Debug level
```

Comment:
Recommend adding a ["count" verbosity arg](https://click.palletsprojects.com/en/7.x/options/#counting) to CLI arguments

Reply:
I agree in the long run. I have a plan for structured logging of the SEIR tasks.  Right now, just trying to make consistent with the forecast tasks.